packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -657,6 +657,7 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
             </Fragment>
         }
         <MIMap
+            ref={(mapsIndoorsInstance) => {window.mapsIndoorsInstance = mapsIndoorsInstance}}
```

Comment:
It feels a little convoluted to write `window.mapsIndoorsInstance.getMapsIndoorsInstance()`. Can we shorten it somehow?